components/inputs/README.md

```diff
@@ -9,88 +9,8 @@ There are various input components available:
 - [Radio Inputs (input-radio-*)](docs/input-radio.md)
 - [Search (input-search)](docs/input-search.md)
 - [Select Lists (input-select-styles)](docs/input-select-styles.md)
-- [Text (input-text, input-styles, input-textarea)](docs/input-text.md)
+- [Text (input-text, input-textarea)](docs/input-text.md)
```

Comment:
This page doesn't appear on the Daylight Site, so I've moved all the relevant information about fieldsets and labels into other places that do.